clang/docs/ReleaseNotes.rst

```diff
@@ -384,6 +384,10 @@ Modified Compiler Flags
   evaluating to ``true`` and an empty body such as ``while(1);``)
   are considered infinite, even when the ``-ffinite-loop`` flag is set.
 
+- Removed "arm interrupt calling convention" warning that was included in
```

Comment:
I would rephrase this a bit:
```
The "arm interrupt calling convention" warning previously included in ``-Wextra`` has been removed and made into a unique warning enabled using ``-Warm-interrrupt-vfp-clobber``.
```

Reply:
"Made into" implies that it's the more or less the same warning, but I'm trying to convey that the existing warning was removed because it's unsound, and the name is part of its confusedness. I will revisit this.